plugins/view-transitions/includes/theme.php

```diff
@@ -27,6 +27,67 @@ function plvt_polyfill_theme_support(): void {
 	add_theme_support( 'view-transitions' );
 }
 
+/**
+ * Sanitizes theme support arguments for the 'view-transitions' feature.
+ *
+ * If the feature was part of WordPress Core, the logic of this function would become part of the `add_theme_support()`
+ * function instead. There is no action or filter that could be used though, hence it is implemented here in a separate
+ * function that runs after `after_setup_theme`, but before the 'view-transitions' feature arguments are possibly used.
+ *
+ * @since 1.0.0
+ *
+ * @global array<string, mixed> $_wp_theme_features Theme support features added and their arguments.
+ */
+function plvt_sanitize_view_transitions_theme_support(): void {
```

Comment:
Idea: Instead of having `plvt_sanitize_view_transitions_theme_support()` sanitize the global variable and then override `$_wp_theme_features` with the sanitized value, what if there was instead a `plvt_get_view_transitions_theme_support()` which always returned the sanitized value? Then there would be no need to set `plvt_sanitize_view_transitions_theme_support()` at `init` action, and there wouldn't be a risk that a theme made changes to the theme support after the `init` action happened.

Reply:
That's a good idea, though I personally think the risk is low enough to ignore it in favor of having a more Core-like API, relying on the actual functions that should be used for this and "patching" the lack of integrated sanitization via a hook like this.

If later there are reports of problems with this, we could reassess the relevance.